app/router/user.py

```diff
@@ -342,3 +342,78 @@ def get_user_profile(
 
     log(log.INFO, "User [%s] info", user.username)
     return user
+
+
+@user_router.patch(
+    "/notification-settings",
+    status_code=status.HTTP_200_OK,
+    response_model=s.User,
+)
+def patch_user_notification_settings(
+    notification_settings: s.UserNotificationSettingsIn,
+    db: Session = Depends(get_db),
+    current_user: m.User = Depends(get_current_user),
+):
+    if notification_settings.notification_job_status is not None:
+        current_user.notification_job_status = (
+            notification_settings.notification_job_status
+        )
+
+    if notification_settings.notification_profession is not None:

```

Comment:
I would also check if length of `notification_settings.notification_profession` is greater then 0?